### PR TITLE
Weather App Stockholm

### PR DESCRIPTION
**Pull request by lunek1**
## Netlify link
https://fanciful-melomakarona-4e6d38.netlify.app/

## Collaborators
Solo project, no collaborators.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <body>
+    <!-- Parent container -->
+    <div id="container">
+      <!-- Weather condition and sunrise/sunset times will be displayed here -->
+      <div id="weather-overview">
+        <p>
+          <span id="weather-condition"></span> | <span id="temperature"></span>
+        </p>
+        <p>sunrise <span id="sunrise-time"></span></p>
+        <p>sunset <span id="sunset-time"></span></p>
```
Comment (vittoriamatteoli) [inline index.html:18]: to try a different approach, this could be implemented in the JS file.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        const timestamp = item.dt * 1000; // Convert seconds to milliseconds
+        const date = new Date(timestamp);
+        const dayOfWeek = date
+          .toLocaleString("en-US", { weekday: "short" })
+          .toLowerCase(); // Get day of the week, short, converted to lower case
+        const temperature = item.main.temp.toFixed(0); // Get temperature
+        return {
+          dayOfWeek,
+          temperature,
+        };
```
Comment (vittoriamatteoli) [inline script.js:101]: nice! :)

**Review by vittoriamatteoli — COMMENTED** [review]
Nice job! The project fulfils the requirement and looks great, you reproduce the style perfectly! I would just delete all folders and files that you don't need.

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Cornelia, very good job this week! Your code is well structured abnd formatted. The app is working as it should, you completed the basic requirements to pass this assignment. You can delete the design files to clean up the project? One comment is about the forecast: 

- The app should have: 4-day forecast You are displaying all 5 days when we requested 4
- the temperature should be rounded to 1 decimal

Not a big deal. Fix these things and you're good to go:) 
Keep up the awesome work! 🌷

**Comment by HIPPIEKICK** [comment]
Can you provide a working Netlify link please 😊

**Comment by lunek1** [comment]
Hi!
This is the link I updated on my github:
https://fanciful-melomakarona-4e6d38.netlify.app/

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by lunek1** [comment]
> Hi Cornelia, very good job this week! Your code is well structured abnd formatted. The app is working as it should, you completed the basic requirements to pass this assignment. You can delete the design files to clean up the project? One comment is about the forecast:
> 
> * The app should have: 4-day forecast You are displaying all 5 days when we requested 4
> * the temperature should be rounded to 1 decimal
> 
> Not a big deal. Fix these things and you're good to go:) Keep up the awesome work! 🌷

**Comment by lunek1** [comment]
> Hi Cornelia, very good job this week! Your code is well structured abnd formatted. The app is working as it should, you completed the basic requirements to pass this assignment. You can delete the design files to clean up the project? One comment is about the forecast:
> 
> * The app should have: 4-day forecast You are displaying all 5 days when we requested 4
> * the temperature should be rounded to 1 decimal
> 
> Not a big deal. Fix these things and you're good to go:) Keep up the awesome work! 🌷

Hi! Can you please check the changes I made and see if I met the requirements now. Also let me know if my netlify link is working for you. ✨

**Comment by AntonellaMorittu** [comment]
Hi Cornelia, the PR is closed and I cannot approve it, please open it again 👍

**Comment by lunek1** [comment]
> Hi Cornelia, the PR is closed and I cannot approve it, please open it again 👍

I didn't even know I had closed. It's open now 😅

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**